app/models/buy_intention.rb

```diff
@@ -0,0 +1,5 @@
+class BuyIntention < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
```

Comment:
why is the name unique?

Reply:
Because it's a person model classification.